test/e2e/specs/editor/various/convert-block-type.spec.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Code block', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
```

Comment:
I think we need to remember to `deleteAllPosts()` in `test.afterEach` if we create one in `test.beforeEach` 😅 (I also forgot this all the time).